frontend/static/js/index.js

```diff
@@ -1,8 +1,7 @@
-import { router, navigateTo } from './router.js';
+import {router, navigateTo} from './router.js';
 import logoutService from './services/logoutService.js';
 
-
-sessionStorage.setItem('isLoggedIn', false);
+// localStorage.setItem('isLoggedIn', false);
```

Comment:
is this by design commented out?

Reply:
Yes and no. I did want it gone though to see whether I corrected everything that depended on the initialisation there (which nothing does anymore)